### DISCUSSION TITLE
disp_float_fmt ignored for showing row

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1706] [Q&A]
When I press Enter on a CSV sheet to see a row as a sheet, it knows to type some values as floats, but why doesn't it use my display format for these floats? I've got `disp_float_fmt` set up to my favorite format but it's ignored in this scenario.

**Comment by saulpw** [comment]
`disp_*_fmt` options are only for columns typed as such.  We could expand the display engine to use the formatters for individual cells too; I can see that being useful.
